### PR TITLE
add admin API to api docs

### LINKED ISSUE #469: Add admin APIs to API doc
Currently missing from 

https://github.com/google/exposure-notifications-verification-server/blob/main/docs/api.md

/kind enhancement

### PR DESCRIPTION
**Pull request by mikehelmick**
Fixes #469 

## Proposed Changes

* Add API docs for admin APIs
*
*

**Release Note**

```release-note
Added documentation for ADMIN APIs
```

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -127,6 +127,111 @@ Possible error code responses. New error codes may be added in future releases.
...
+```
+
+* `sypmtomDate` and `testDate` are both optional.
+  * only one will be encoded into the eventually issued certificate
+  * symptom date is always preferred to test date
+* `testType`
+  * Must be `confirmed`, `likely`, `negative`
+  * valid values depends on your realm's settings
+* `tzOffset`
+  * Offset in minutes of the user's timezone. Positive, negative, 0, or omitted (using the default of 0) are all valid. 0 is considered to be UTC.
```
Comment (icco) [inline docs/api.md:155]: Minutes? Isn't this usually hours?

**Review by icco — APPROVED** [review]
/lgtm
/hold for one small question. Feel free to submit.

File: docs/api.md [hunk 2]
```diff
@@ -127,6 +127,111 @@ Possible error code responses. New error codes may be added in future releases.
...
+
+These APIs are available on the admin server and require and `ADMIN` level API key.
+
+## `/api/issue`
+
+Request a verification code to be issued. Accepts [optional] symptom date and test dates in ISO 8601 format. These can be in local time, if a timezone offset is provided. If a phone number is provided and the realm is configured with SMS credentials, then an SMS will be dispatched according to the realm's settings.
+
+```json
+{
+	"symptomDate": "YYYY-MM-DD",
```
Comment (sethvargo) [inline docs/api.md:140]: Nit: 2 spaces on the json

File: docs/api.md [hunk 3]
```diff
@@ -127,6 +127,111 @@ Possible error code responses. New error codes may be added in future releases.
...
+	"testDate": "YYYY-MM-DD",
+	"testType": "<valid test type>",
+	"tzOffset": 0,
+	"phone": "+1NPANXXLINE",
```
Comment (sethvargo) [inline docs/api.md:144]: I'm not sure I understand this format?

File: docs/api.md (hunk 1 above)
Comment (mikehelmick) [reply docs/api.md:155]: this is the way it is in the api - not all TZ offsets are even hour increments

File: docs/api.md (hunk 3 above)
Comment (mikehelmick) [reply docs/api.md:144]: made clearer - was https://en.wikipedia.org/wiki/North_American_Numbering_Plan#Modern_plan

**Pull request merged**